### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by ZjeniiaSt**
### First task 

[Demo](https://zjeniiast.github.io/kottans-popup/) |
[Code base](https://github.com/ZjeniiaSt/kottans-popup)

### CODE REVIEW

**Comment by ZjeniiaSt** [comment]
I think I should explain something. I tried to change commits in history and deleted too many. The previous request was closed automatically (. And when restoring commits - git offered to create a new request.

File: submissions/ZjeniiaSt/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+              <a href="#">
+                <img src="./images/194938.png" />
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+  </body>
+</html>
```
Comment (artem-trubin) [inline submissions/ZjeniiaSt/html-css-popup/index.html:146]: Add empty lines at the end of each file. I believe you can set your VSCode to do so for you each time you save a file.

File: submissions/ZjeniiaSt/html-css-popup/styles.css [hunk 2]
```diff
@@ -0,0 +1,207 @@
+
```
Comment (artem-trubin) [inline submissions/ZjeniiaSt/html-css-popup/styles.css:1]: No need for an empty line at the start.

File: submissions/ZjeniiaSt/html-css-popup/styles.css [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    top: 100px;
+    right: 10%;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .menu {
+    right: 8%;
+  }
+}
```
Comment (artem-trubin) [inline submissions/ZjeniiaSt/html-css-popup/styles.css:207]: Another empty line.

**Review by artem-trubin — COMMENTED** [review]
Very clean work!

Just a UX comment: your type of showing the currently focused button isn't very user-friendly. A small horizontal scale visually shows that you've jumped to the next button at the time you are navigating the menu, but it's not enough to make a distinct difference among other buttons and if you look away for a while and then take a glance at the buttons, it's hard to understand what exactly is currently focused. A highlighted background, shadow, or outline is better in these situations. But that's just for the future. Check the comments below to fix some code and we're good to go here.

**Comment by ZjeniiaSt** [comment]
ok, i'll do it. But I don't understand,  why this lines must be added....
How it chang the code...

сб, 20 серп. 2022, 09:51 користувач Artem Trubin ***@***.***>
пише:

**Comment by artem-trubin** [comment]
You can read about empty lines in the end of files [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

**Comment by ZjeniiaSt** [comment]
thanks

сб, 20 серп. 2022, 11:02 користувач Artem Trubin ***@***.***>
пише:

**Pull request merged**